jenkins/poc/haim/run_policies.py

```diff
@@ -0,0 +1,21 @@
+
+import os
+
+
+AWS_ACCESS_KEY_ID_APPENG = os.environ['AWS_ACCESS_KEY_ID_DELETE_PERF']
+AWS_SECRET_ACCESS_KEY_APPENG = os.environ['AWS_SECRET_ACCESS_KEY_DELETE_PERF']
+LDAP_HOST_NAME = os.environ['LDAP_HOST_NAME']
+GOOGLE_APPLICATION_CREDENTIALS = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
+SPREADSHEET_ID = os.environ['AWS_IAM_USER_SPREADSHEET_ID']
+
+
+LOGS = os.environ.get('LOGS', 'logs')
+
+mandatory_tags_appeng = {'Budget': 'APPENG'}
+
+os.system(f"""echo "Running the tag_iam_user" """)
+os.system(f"""podman run --rm --name cloud-governance-poc-haim --net="host" -e account="APPENG" -e policy="tag_iam_user" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_APPENG}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_APPENG}" -e user_tag_operation="update" -e SPREADSHEET_ID="{SPREADSHEET_ID}" -e GOOGLE_APPLICATION_CREDENTIALS="{GOOGLE_APPLICATION_CREDENTIALS}" -v "{GOOGLE_APPLICATION_CREDENTIALS}":"{GOOGLE_APPLICATION_CREDENTIALS}" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}"  -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
+
+
+os.system(f"""echo "Running the tag_resources" """)
```

Comment:
Is that including only tag resource ?
I ask because we need to run the policies in dry run = "yes" ?

Reply:
yes, it tags the resources.